Can Python2 be supported?

I'm on CentOS 8.1.x ThunderX2 and see:

```

[ 14%] Built target scopmath
[ 15%] Generating kinderiv.h by inspecting MOD files
/bin/sh: python: command not found
make[2]: *** [coreneuron/CMakeFiles/kin_deriv_header.dir/build.make:88: coreneuron/_kinderiv.h] Error 127
make[1]: *** [CMakeFiles/Makefile2:1046: coreneuron/CMakeFiles/kin_deriv_header.dir/all] Error 2
make: *** [Makefile:183: all] Error 2
```

yum info python2 tells me it is there.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -87,6 +87,11 @@ include(OpenAccHelper)
 include(ClangFormatHelper)
 
 # =============================================================================
+# Common dependencies
+# =============================================================================
+find_package(PythonInterp REQUIRED)
+
+# =============================================================================
 # ISPC should compile with --pic by default
 # =============================================================================
 set(CMAKE_ISPC_FLAGS "--pic ${CMAKE_ISPC_FLAGS}")

--- a/extra/nrnivmodl_core_makefile.in
+++ b/extra/nrnivmodl_core_makefile.in
@@ -190,7 +190,7 @@ $(kinderiv_h): $(mod_all_cpp_files) build_always | $(MODC_DIR)
 	cd $(MODC_DIR) && mkdir -p _tmp &&	mv [^_]*.cpp _tmp/ && \
 	  mv $(addprefix _tmp/,$(notdir $(mod_all_cpp_files))) ./ || true
 	rm -rf $(MODC_DIR)/_tmp
-	cd $(MODC_DIR) && python $(datadir)/kinderiv.py
+	cd $(MODC_DIR) && @PYTHON_EXECUTABLE@ $(datadir)/kinderiv.py
 
 
 $(dimplic_c): $(dimplic_src) | $(MODC_DIR)

